imagery/i.ortho.photo/lib/find_init.c

```diff
@@ -5,14 +5,21 @@
  **************************************************************/
 #include <grass/gis.h>
 
+#define ELEMENT_BUFFER_SIZE 80
+
 int I_find_initial(char *group)
 {
     char *element;
+    int file_exists;
 
-    element = (char *)G_malloc(80 * sizeof(char));
-
-    if (group == NULL || *group == 0)
+    if (group == NULL || *group == 0) {
         return 0;
-    sprintf(element, "group/%s", group);
-    return G_find_file(element, "INIT_EXP", G_mapset()) != NULL;
+    }
```

Comment:
Please remove the `{}` which was not originally there and there is no need to add it (to have easier to read diff).

Reply:
Done